cloudbuild_CI.yaml

```diff
@@ -31,6 +31,13 @@ steps:
       - 'push'
       - 'gcr.io/${PROJECT_ID}/gcp-variant-transforms:${COMMIT_SHA}'
   # Run the test script from the image we just built and pushed.
+  #
+  # Note that the only reason that we use the built image in the 'name' argument
```

Comment:
Thanks, Bashir! This makes sense now!
Consider rephrasing to be more concise. Something like:
Note that './deploy_and_run_tests.sh' actually references the source downloaded from GitHub and not a file in the docker image we just built. The only reason that we use the built image in the 'name' argument is to avoid reinstalling python, virtualenv, etc on the image.